Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -298,13 +290,37 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
 - (void)toxAV:(OCTToxAV *)toxAV audioBitRateChanged:(OCTToxAVAudioBitRate)bitrate stable:(BOOL)stable friendNumber:(OCTToxFriendNumber)friendNumber
 {
-    // Lower bitrate if unstable?
+    if (stable) {
+        return;
+    }
+
+    OCTToxAVAudioBitRate newBitrate;
+
+    switch (bitrate) {
+        case OCTToxAVAudioBitRateDisabled:
+            NSAssert(NO, @"We shouldn't be here!");
+            break;
+        case OCTToxAVAudioBitRate8:
+            return;
+        case OCTToxAVAudioBitRate48:
+            newBitrate = OCTToxAVAudioBitRate32;
+            break;
+        case OCTToxAVAudioBitRate32:
+            newBitrate = OCTToxAVAudioBitRate24;
+            break;
+        case OCTToxAVAudioBitRate24:
+            newBitrate = OCTToxAVAudioBitRate16;
+            break;
+        case OCTToxAVAudioBitRate16:
+            newBitrate = OCTToxAVAudioBitRate8;
+            break;
+    }
```

Comment:
This looks okay, you can just change the ordering to be more consistent. From lowest to highest.